interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
```

Comment:
@mhuyck: Thank you for the clarification. The reason why I don't feel comfortable about the *.spec.js" is because in JavaScript test framework, `spec` is a synonym of "test cases", but this file includes a module instead of test case(s).

There are some interesting discussions online about DRY vs. DAMP when writing test code. For example:
http://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests
I would rather have codeclimate exclude the spec files in order to keep the unit tests really UNIT. Considering the upcoming deadline, I am going to create an issue for further discussion in the future.